lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -431,11 +431,22 @@ TypeSystemSwiftTypeRef::GetClangTypeNode(CompilerType clang_type,
   return pointee ? GetPointerTo(dem, nominal) : nominal;
 }
 
-/// Return a pair of modulename, type name for the outermost nominal type.
-static std::optional<std::pair<StringRef, StringRef>>
-GetNominal(swift::Demangle::Demangler &dem, swift::Demangle::NodePointer node) {
+bool 
+TypeSystemSwiftTypeRef::IsBuiltinType(CompilerType type) {
```

Comment:
Maybe call this IsSwiftBuiltinType?
alternatively — is this different from CmpilerType.IsIntegerType() || CmpilerType.IsFloatingPointType() || CmpilerType.IsVectorType()

Reply:
Is this even used in the patch?

Reply:
This function is unrelated, the diff is probably highlighting it because I changed code above and below it?